client/src/components/ShrinkImage/index.tsx

```diff
@@ -15,10 +15,13 @@ const styles = (theme: Theme) =>
     shrinkimage: {
       display: 'block',
       height: '100%',
-      maxWidth: '80vw',
-      maxHeight: '80vh',
+      maxWidth: '75vw',
+      maxHeight: '75vh',
       margin: `${theme.spacing(4)}px auto`,
       transition: 'transform 300ms ease-in-out',
+      [theme.breakpoints.down('sm')]: {
+        height: 'auto',
+      },
```

Comment:
On desktop we can control the size of the image by setting the height of the containing div. We can make it bigger or smaller without hitting the maxWidth and messing up the width/height ratio.

On mobile we let the image scale down appropriately to fit 75% the width of the screen.

Reply:
👍